.github/workflows/build.yml

```diff
@@ -253,13 +271,108 @@ jobs:
+        name: translateLocally.latest.${{ matrix.os }}.${{ matrix.arch }}.deb
         path: ${{github.workspace}}/build/deb/*.deb
 
+  build-windows:
+    strategy:
+      matrix:
+        include:
+          - name: "Windows 2019 avx"
+            os: windows-2019
+            arch: "core-avx-i"
+          - name: "Windows 2022 avx"
+            os: windows-2022
+            arch: "core-avx-i"
+          - name: "Windows 2019 avx2"
+            os: windows-2019
+            arch: "core-avx2"
+          - name: "Windows 2022 avx2"
+            os: windows-2022
+            arch: "core-avx2"
+      fail-fast: false
+
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.name }}
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        
+    - name: Download MKL
+      run: |
+        # Wget retries downloading files and is faster than Invoke-WebRequest
+        C:\msys64\usr\bin\wget.exe -nv ${{ env.MKL_URL }} -O mkl.zip
+        Expand-Archive -Force mkl.zip ${{ github.workspace }}\mkl
+        # Set MKLROOT environment variable so that CMake can find MKL
+        echo "MKLROOT=${{ github.workspace }}\mkl" | Out-File -FilePath $env:GITHUB_ENV  -Encoding utf8 -Append
+      shell: powershell
+    
+    - name: Cache vcpkg
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-vcpkg
+      with:
+        path: C:\vcpkg # included with the installation #@TODO get vcpkg git ID here?
+        key: ${{ matrix.os }}-build-${{ env.cache-name }}
+        restore-keys: |-
+            ${{ matrix.os }}-build-${{ env.cache-name }}
+            ${{ matrix.os }}-build
+
+    - name: Install dependencies with vcpkg
+      working-directory: C:\vcpkg
+      run: |
+        $Env:VCPKG_BUILD_TYPE = 'release'
+        $Env:VCPKG_DEFAULT_TRIPLET = 'x64-windows-static' # QT6 version, linguist tools not working yet: qtbase:x64-windows-static qttools:x64-windows-static qtsvg:x64-windows-static
+        .\vcpkg install protobuf:x64-windows-static pcre2:x64-windows-static zlib:x64-windows-static libarchive:x64-windows-static qt5-base:x64-windows-static qt5-tools:x64-windows-static qt5-svg:x64-windows-static
+      shell: powershell
+        
+    - name: Fix PCRE2 extensions # https://github.com/ugermann/ssplit-cpp/pull/29
+      working-directory: C:\vcpkg
+      run: |
+        cd installed/x64-windows-static/lib
+        # the ssplit-cpp finds staticd libs only because the extension is hardcoded to only find the debug
+        # version of the static lib: https://github.com/browsermt/ssplit-cpp/blob/master/cmake/FindPCRE2.cmake#L66
+        # or it could be a number of other reasons. Luckily we have the power of cp to rule the universe.
+        cp pcre2-8-static.lib pcre2-8.lib
+        cp pcre2-8-static.lib pcre2.lib
+        cp pcre2-16-static.lib pcre2-16.lib
+        cp pcre2-32-static.lib pcre2-32.lib
+        cp pcre2-posix-static.lib pcre2-8.lib
```

Comment:
Is this something we should eventually fix upstream? I noticed there [was already something happening to that line](https://github.com/browsermt/ssplit-cpp/pull/10). Since its just bunch of search names, should be fairly easy to search for `-static` or `-staticd` based on `CMAKE_RELEASE_TYPE`.

Reply:
I think that Jerin's PR would break compilation even further. A (somewhat improper) fix would be https://github.com/ugermann/ssplit-cpp/pull/29
We shouldn't ever link to a debug PCRE2, if we have a crash in our code, we don't need PCRE2's debug symbols...

Reply:
You're right. Probably better to go that route then. I'd assume anyone who has debug libraries installed also has the non-debug versions available. If not, they're probably capable enough to fix that on their own.